sql/load_demo_data.sql

```diff
@@ -2981,10 +2981,10 @@ UNLOCK TABLES;
 
 LOCK TABLES `oh_medicaldsrtype` WRITE;
 /*!40000 ALTER TABLE `oh_medicaldsrtype` DISABLE KEYS */;
-INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('D','Drugs',NULL,NULL,NULL,NULL,1);
-INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('K','Chemical',NULL,NULL,NULL,NULL,1);
-INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('L','Laboratory',NULL,NULL,NULL,NULL,1);
-INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('S','Surgery',NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED` `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('D','Drugs','N',NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED` `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('K','Chemical','N',NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED` `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('L','Laboratory','N',NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED` `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('S','Surgery','N',NULL,NULL,NULL,NULL,1);
```

Comment:
Please always do a smoke test fo the changes:

```suggestion
INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('D','Drugs','N',NULL,NULL,NULL,NULL,1);
INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('K','Chemical','N',NULL,NULL,NULL,NULL,1);
INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('L','Laboratory','N',NULL,NULL,NULL,NULL,1);
INSERT INTO `oh_medicaldsrtype` (`MDSRT_ID_A`, `MDSRT_DESC`, `MDSRT_DELETED`, `MDSRT_CREATED_BY`, `MDSRT_CREATED_DATE`, `MDSRT_LAST_MODIFIED_BY`, `MDSRT_LAST_MODIFIED_DATE`, `MDSRT_ACTIVE`) VALUES ('S','Surgery','N',NULL,NULL,NULL,NULL,1);
```